types/knex/index.d.ts

```diff
@@ -307,6 +307,11 @@ declare namespace Knex {
 
     interface WhereIn {
         (columnName: string, values: Value[]): QueryBuilder;
+        (columnNames: string[], values: Value[][]): QueryBuilder;
+        (columnName: string, subquery: QueryBuilder): QueryBuilder;
+        (columnNames: string[], subquery: QueryBuilder): QueryBuilder;
+        (columnName: string, callback: QueryCallback): QueryBuilder;
+        (columnNames: string[], callback: QueryCallback): QueryBuilder;
```

Comment:
Could this also be expressed like this?
`(columnNames: string|string[], callback: Value[]|Value[][]|QueryBuilder|QueryCallback): QueryBuilder;`

I'm not sure which way is preferred.

Reply:
How about:
```typescript
(columnName: string, value: Value[] | QueryBuilder | QueryCallback): QueryBuilder
(columnNames: string[], values: Value[][] | QueryBuilder | QueryCallback): QueryBuilder
```

Reply:
⚡️